models/world_aiddata_gcdf/code/upload.py

```
@@ -0,0 +1,101 @@
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--table", default=None, help="Upload only this table")
+    args = parser.parse_args()
+
+    tables = [args.table] if args.table else list(EXPECTED_ROWS)
+    for table_id in tables:
+        try:
+            upload_table(table_id)
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)